logout.php

```diff
@@ -2,12 +2,25 @@
 require_once __DIR__ . '/vendor/autoload.php';
 
 session_start();
+Dotenv::load(__DIR__);
```

Comment:
We don't need to load .env etc also as stated with `NullAdapter`. 

Thank you.